tests/test_key_locator.py

```diff
@@ -0,0 +1,253 @@
+import asyncio
+import os
+
+import pytest
+import requests
+from api_test_utils.apigee_api_apps import ApigeeApiDeveloperApps
+from api_test_utils.apigee_api_products import ApigeeApiProducts
+
+JWKS_URL = "https://raw.githubusercontent.com/NHSDigital/identity-service-jwks" \
+    "/main/jwks/internal-dev/278af795-6884-42dd-a713-f2b03927abb5.json"
+ENVIRONMENT = os.environ["APIGEE_ENVIRONMENT"]
+BASE_PATH = os.environ["SERVICE_BASE_PATH"]
+
+
+@pytest.fixture(scope="module")
+def event_loop():
+    loop = asyncio.get_event_loop()
+    yield loop
+    loop.close()
+
+
+@pytest.fixture(scope='module')
+def apigee_root_url():
+    return "https://api.service.nhs.uk" if ENVIRONMENT == "prod" \
```

Comment:
If we really wanted to avoid breaking the line here, we could calculate the prexix first then stick it all together at the end:
```python
prefix = "" if ENVIRONMENT == "prod" else f"{ENVIRONMENT}."
return f"https://{PREFIX}api.service.nhs.uk"
```